app/views/profile/orders/index.html.erb

```diff
@@ -2,12 +2,12 @@
 <section class="orders">
   <% @user.orders.each do |order| %>
     <section id="order-<%= order.id %>">
-      <h3>Order ID: <%= link_to order.id, "/profile/orders/#{order.id}" %></h3>
-      <p>Order Created On: <%= order.created_at %><br>
-      Order Updated On: <%= order.updated_at %><br>
-      Order Status: <%= order.status %><br>
-      Total Quantity of Items: <%= order.total_quantity %><br>
-      Grand Total: <%= order.grandtotal %></p>
+      <p>Order ID: <%= link_to order.id, "/profile/orders/#{order.id}" %></p>
```

Comment:
If you can update the styling to use the h3 and breaks I had, I would appreciate it, but understand if you don't want to.